chacractl/api/binaries.py

```diff
@@ -136,6 +136,8 @@ def main(self):
                 self.post(url, filepath)
 
         elif self.parser.has('delete'):
+            if self.parser.get('delete') is None:
```

Comment:
Actually, when I change it to `elif self.parser.get('delete'):`, then it just silently exits. I think we should keep it as is, so it's more user-friendly?